web/themes/new_weather_theme/templates/partials/alerts-in-hourly-table.html.twig

```diff
@@ -0,0 +1,20 @@
+{% set periodLength = periods | length %}
+{% for alertInfo in alertPeriods %}
+  {% set remainingCount = periodLength - alertInfo.duration - alertInfo.periodIndex %}
+  <tr data-row-name="alert">
+    <th scope="row" class="position-sticky left-0"><span class="usa-sr-only">{{ "Alert" | t }}</span></th>
+    {% if alertInfo.periodIndex > 0 %}      
+      {% for i in 1..alertInfo.periodIndex %}
+        <td></td>
+      {% endfor %}
```

Comment:
Could this be done with just a colspan? (Genuine question, I dunno if a colspan would screw something up.)

```suggestion
      <td colspan="{{ alertInfo.periodIndex }}"></td>
```

Reply:
This is...so obvious....